root/defaults/proxy.conf

```diff
@@ -14,10 +14,13 @@ proxy_connect_timeout 240;
 proxy_set_header Host $host:$server_port;
 proxy_set_header X-Real-IP $remote_addr;
 proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-proxy_set_header X-Forwarded-Proto https;
+proxy_set_header X-Forwarded-Proto $scheme;
+proxy_set_header X-Forwarded-Host $host;
+proxy_set_header X-Forwarded-Ssl on;
 proxy_redirect  http://  $scheme://;
 proxy_http_version 1.1;
 proxy_set_header Connection "";
+proxy_cookie_path / "/; HTTPOnly; Secure";
```

Comment:
This seems to be a pretty crude approach. Likely to break certain apps who rely on cookies not meant to be httponly. This should be added to specific reverse proxy configs on a case by case basis

Reply:
I believe just having `secure` in there satisfies the scanner (I'll have to test). Would it be acceptable to keep this line but omit `httponly`?